### PR TITLE
Prepare 0.7 release

### LINKED ISSUE #553: Release 0.7
There are several bug fixes and general usability improvements in [here](https://github.com/amethyst/amethyst/compare/release-0.6...develop).  I feel this warrants releasing a 0.7 version.

I'll plan on doing this sometime in early February unless there are dissenting opinions on doing a release.

### LINKED ISSUE #677: 0.7 release
Hi! 

I just discovered amethyst, and I would really like to start making games with it. 

A lot of things have changed in the engine recently and it seems to me that a lot of things in the code are new compared to the early January release. 

(I hope I am posting my question in the good place).

### PR DESCRIPTION
**Pull request by Xaeroxe**
Closes #677 
Closes #553 

@Rhuagh I also added the note you requested to README.md

---
This change is [Reviewable](https://reviewable.io/reviews/amethyst/amethyst/730)

### CODE REVIEW

**Comment by Xaeroxe** [comment]
@jojolepro I know you're busy with exams (good luck!) so feel free to review or ignore this as time permits.

File: README.md [hunk 1]
```diff
@@ -86,6 +80,18 @@ $ sudo apt install libasound2-dev libx11-xcb-dev
 $ sudo yum install alsa-lib-devel
 ```
 
+## OSX audio workaround
```
[the remark this answers is not in the document]
Comment (torkleyy) [reply README.md]: There's another person with permission to release, I commented on the issue.

File: README.md [hunk 2]
```diff
@@ -19,8 +19,6 @@
 
 **Warning: The book and the documentation are missing content. Amethyst is undergoing a lot of changes at the moment.**
 
-This project is a *work in progress* and is very incomplete; pardon the dust!
-
 ## Goals
 
 * massively parallel architecture
```
Comment (jojolepro) [inline README.md]: Should start with an uppercase to follow how the other bullet points of the book were made.

**Comment by Xaeroxe** [comment]
Blocked on resolution of https://github.com/amethyst/amethyst/issues/731

**Pull request merged**